pkgs/development/python-modules/python-bip32/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pytestCheckHook
+, pythonOlder
+, base58
+, coincurve
+}:
+
+buildPythonPackage rec {
+  pname = "python-bip32";
```

```suggestion
  pname = "bip32";
```
pname should match the library's name on PyPI.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines